rero_invenio_base/cli/es/index.py

```
@@ -29,27 +28,58 @@ def _create_index_from_mapping(index_name, f_mapping):
+    """Fetch current task status and print a one-line progress update.
+
+    :param task: task id.
+    :param count: elapsed seconds to display.
+    """
+    try:
+        res = current_search_client.tasks.get(task)
+        status = res.get("task", {}).get("status", {})
+        created = status.get("created", "?")
+        total = status.get("total", "?")
+        click.secho(f"Watching task: {task} {count} seconds ... {created}/{total}", fg="green")
+    except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)